src/absil/ilread.fs

```diff
@@ -3970,15 +3970,16 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-    if IL.runningOnWindows then
```

Comment:
remove the changes in this file thanks (prefer what is in the FCS repository please)